packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -72,266 +79,136 @@ class _DeepLinkListViewMainPanel extends StatelessWidget {
-                style: Theme.of(context).textTheme.bodyLarge,
-              ),
-              actions: [
-                SizedBox(
-                  width: wideSearchFieldWidth,
-                  child: DevToolsClearableTextField(
-                    labelText: '',
-                    hintText: 'Search a URL, domain or path',
-                    prefixIcon: const Icon(Icons.search),
-                    onChanged: (value) {
-                      controller.searchContent = value;
-                    },
-                  ),
+
+    return ValueListenableBuilder<DisplayOptions>(
+      valueListenable: controller.displayOptionsNotifier,
+      builder: (context, displayOptions, _) =>
+          ValueListenableBuilder<List<LinkData>?>(
+        valueListenable: controller.allLinkDatasNotifier,
+        builder: (context, linkDatas, _) {
+          if (linkDatas == null) {
+            return const CenteredCircularProgressIndicator();
+          }
+          if (displayOptions.showSplitScreen) {
+            return Row(
+              children: [
+                Expanded(
+                  child: _AllDeepLinkDataTable(controller: controller),
                 ),
-              ],
-            ),
-            const SizedBox(height: denseSpacing),
-            const TabBar(
-              tabs: [
-                Text('Domain view'),
-                Text('Path view'),
-                Text('Single URL view'),
-              ],
-              tabAlignment: TabAlignment.start,
-              isScrollable: true,
-            ),
-            Expanded(
-              child: ValueListenableBuilder<bool>(
-                valueListenable: controller.showSpitScreenNotifier,
-                builder: (context, showSpitScreen, _) => TabBarView(
-                  children: [
-                    _DataTableWithValidationDetails(
-                      tableView: TableViewType.domainView,
-                      linkDatas: controller.getLinkDatasByDomain,
-                      controller: controller,
-                      showSpitScreen: showSpitScreen,
-                    ),
-                    _DataTableWithValidationDetails(
-                      tableView: TableViewType.pathView,
-                      linkDatas: controller.getLinkDatasByPath,
-                      controller: controller,
-                      showSpitScreen: showSpitScreen,
-                    ),
-                    _DataTableWithValidationDetails(
-                      tableView: TableViewType.singleUrlView,
-                      linkDatas: linkDatas,
-                      controller: controller,
-                      showSpitScreen: showSpitScreen,
+                Expanded(
+                  child: ValueListenableBuilder<LinkData?>(
+                    valueListenable: controller.selectedLink,
+                    builder: (context, selectedLink, _) => TabBarView(
+                      children: [
+                        ValidationDetailScreen(
```

Comment:
rename to `ValidationDetailsView` as "Screen" is a name in DevTools reserved for the top level devtools pages